main.go

```diff
@@ -25,11 +23,7 @@ func main() {
 	opts := &plugin.ServeOpts{ProviderFunc: env0.Provider(version)}
 
 	if debugMode {
-		err := plugin.Debug(context.Background(), "registry.terraform.io/env0/env0", opts)
-		if err != nil {
-			log.Fatal(err.Error())
-		}
-		return
+		opts.Debug = true
```

Comment:
can you explain more about this change compared to what we had before?

Reply:
The old code has been deprecated.
I reviewed the documentation, and this is what they suggest doing instead.